definitions/tools/biscuit_align.cwl

```diff
@@ -3,22 +3,32 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Biscuit: align"
-baseCommand: ["/usr/bin/biscuit","align"]
+baseCommand: ["/bin/bash","biscuit_align.sh"]
 requirements:
     - class: ShellCommandRequirement
     - class: ResourceRequirement
       ramMin: 32000
       coresMin: 12
     - class: DockerRequirement
-      dockerPull: "mgibio/bisulfite:v1.3"
+      dockerPull: "mgibio/biscuit:0.3.8"
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'biscuit_align.sh'
+        entry: |
+            set -eo pipefail
+
+            cores=$1
+            outdir=$2
+            read_group_id="$3"
+            reference_index=$4
+            fastq1=$5
+            fastq2=$6
```

Comment:
Might be good to quote filepaths, too, just to be safe.  (Although IIRC this might be one of the tools that doesn't handle that anyway.)